<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qt Plugin Generator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="twPages">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Generator</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="lNameSpace">
          <property name="text">
           <string>Name space</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="leNameSpace"/>
        </item>
        <item row="0" column="2" rowspan="4">
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="lPluginGroup">
          <property name="text">
           <string>Plugin group</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QLineEdit" name="lePluginGroup"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lClassName">
          <property name="text">
           <string>class name</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="leClassName"/>
        </item>
        <item row="1" column="3">
         <widget class="QLabel" name="lPluginIconFilePath">
          <property name="text">
           <string>Plugin icon file path</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QLineEdit" name="lePluginIconFilePath"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="lInheriting">
          <property name="text">
           <string>inherits</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="leInheriting"/>
        </item>
        <item row="2" column="3">
         <widget class="QLabel" name="lPluginToolTip">
          <property name="text">
           <string>Plugin tool tip</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QLineEdit" name="lePluginToolTip"/>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="cbInheritingType"/>
        </item>
        <item row="4" column="0" colspan="5">
         <layout class="QGridLayout" name="gridLayout">
          <item row="3" column="1">
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QToolButton" name="tbRemove">
            <property name="icon">
             <iconset theme="list-remove">
              <normaloff/>
             </iconset>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <widget class="QToolButton" name="tbAdd">
            <property name="icon">
             <iconset theme="list-add">
              <normaloff/>
             </iconset>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QToolButton" name="tbUp">
            <property name="arrowType">
             <enum>Qt::UpArrow</enum>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QToolButton" name="tbDown">
            <property name="text">
             <string>...</string>
            </property>
            <property name="arrowType">
             <enum>Qt::DownArrow</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0" rowspan="7">
           <widget class="QTreeWidget" name="twMembers">
            <property name="editTriggers">
             <set>QAbstractItemView::AllEditTriggers</set>
            </property>
            <property name="rootIsDecorated">
             <bool>false</bool>
            </property>
            <property name="uniformRowHeights">
             <bool>true</bool>
            </property>
            <property name="allColumnsShowFocus">
             <bool>true</bool>
            </property>
            <property name="expandsOnDoubleClick">
             <bool>false</bool>
            </property>
            <attribute name="headerCascadingSectionResizes">
             <bool>true</bool>
            </attribute>
            <attribute name="headerStretchLastSection">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Preview</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTabWidget" name="twPreview">
          <property name="tabPosition">
           <enum>QTabWidget::West</enum>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="wHeader">
           <attribute name="title">
            <string>Header</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QPlainTextEdit" name="pteHeader">
              <property name="font">
               <font>
                <family>Ubuntu Mono</family>
               </font>
              </property>
              <property name="tabChangesFocus">
               <bool>true</bool>
              </property>
              <property name="undoRedoEnabled">
               <bool>false</bool>
              </property>
              <property name="lineWrapMode">
               <enum>QPlainTextEdit::NoWrap</enum>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="tabStopDistance">
               <number>40</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="wSource">
           <attribute name="title">
            <string>Source</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QPlainTextEdit" name="pteSource">
              <property name="font">
               <font>
                <family>Ubuntu Mono</family>
               </font>
              </property>
              <property name="tabChangesFocus">
               <bool>true</bool>
              </property>
              <property name="undoRedoEnabled">
               <bool>false</bool>
              </property>
              <property name="lineWrapMode">
               <enum>QPlainTextEdit::NoWrap</enum>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="tabStopDistance">
               <number>40</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="wPluginHeader">
           <attribute name="title">
            <string>Plugin Header</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QPlainTextEdit" name="ptePluginHeader">
              <property name="font">
               <font>
                <family>Ubuntu Mono</family>
               </font>
              </property>
              <property name="tabChangesFocus">
               <bool>true</bool>
              </property>
              <property name="undoRedoEnabled">
               <bool>false</bool>
              </property>
              <property name="lineWrapMode">
               <enum>QPlainTextEdit::NoWrap</enum>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="tabStopDistance">
               <number>40</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="wPluginSource">
           <attribute name="title">
            <string>Plugin Source</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QPlainTextEdit" name="ptePluginSource">
              <property name="font">
               <font>
                <family>Ubuntu Mono</family>
               </font>
              </property>
              <property name="tabChangesFocus">
               <bool>true</bool>
              </property>
              <property name="undoRedoEnabled">
               <bool>false</bool>
              </property>
              <property name="lineWrapMode">
               <enum>QPlainTextEdit::NoWrap</enum>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="tabStopDistance">
               <number>40</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="wProjectFile">
           <attribute name="title">
            <string>Project Include File</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QPlainTextEdit" name="pteProjectFile">
              <property name="font">
               <font>
                <family>Ubuntu Mono</family>
               </font>
              </property>
              <property name="tabChangesFocus">
               <bool>true</bool>
              </property>
              <property name="undoRedoEnabled">
               <bool>false</bool>
              </property>
              <property name="lineWrapMode">
               <enum>QPlainTextEdit::NoWrap</enum>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="tabStopDistance">
               <number>40</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="dbbButtons">
      <property name="standardButtons">
       <set>QDialogButtonBox::Apply|QDialogButtonBox::Close|QDialogButtonBox::Save</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
